pkg/capacityscheduling/capacity_scheduling.go

```diff
@@ -125,6 +125,7 @@ func New(ctx context.Context, obj runtime.Object, handle framework.Handle) (fram
 		podLister:         handle.SharedInformerFactory().Core().V1().Pods().Lister(),
 		pdbLister:         getPDBLister(handle.SharedInformerFactory()),
 	}
+	logger := klog.FromContext(ctx).WithName(Name).WithValues("Phase", "New")
```

Comment:
To ease reviewing, I'm inclined to keep this PR merely as a "migration" PR - i.e., keep new bits in a follow-up, only migrate klog.InfoS / klog.ErrorS.

Reply:
done